src/python/grpcio/grpc/_cython/_cygrpc/grpc_string.pyx.pxi

```diff
@@ -15,7 +15,6 @@
 import logging
 
 _LOGGER = logging.getLogger(__name__)
-_LOGGER.addHandler(logging.NullHandler())
```

Comment:
Github nit: This shouldn't be here, and it's not actually a change from master - likely just an issue with Github's UI due to branching off an older commit, which should be taken care of automatically when this is merged...but having the file show up here at all is strange

Reply:
Oh my... This PR is too old, I rebased the commit to make this line disappear.